dxp/how_clay_arrives_to_dxp.md

```diff
@@ -0,0 +1,223 @@
+
+| Import          | Description                                                 |
+| --------------- | ----------------------------------------------------------- |
+| base            | All the functionalities of Clay with Bootstrap style        |
+| base-variables  | The main functions and variables of Clay in Bootstrap style |
+| atlas           | All the functionalities of Clay with Lexicon style          |
+| atlas-variables | The main functions and variables of Clay in Lexicon style   |
+
+[Learn everything about Clay CSS](https://github.com/liferay/liferay-frontend-guidelines/blob/281b28e93db2ade5a18230be8ac3b618f28cddb0/css/clay-css-3-intro.md)
+
+---
+
+There is a fifth _unofficial_ line that allow us to include a specific component's CSS we don't usually use it but if you are building something special you can use:
+
+```scss
+@import 'path-to-node_modules-clay-css/components/_buttons'; //not working example
+```
+
+## Which options do we have with Clay?
+
+There are 2 main ways to use Clay in our CSS.
+
+The first one is "read mode" where we can access to variables, mixins and functions of the framework
+
+Among the many variables available in Clay we'd like to highlight:
+
+- Color variables: `$primary`, `$secondary`, `$success`, `$success-l1`, ... 
+- Spacing variables: `$spacers`, `$grid-breakpoints`, `$container-max-widths`, ...
+- Typography variables `$font-family-serif`, `$h1-font-size`, `$display1-size`, ...
+
+Sources: [BS](https://github.com/liferay/clay/blob/master/packages/clay-css/src/scss/bootstrap/_variables.scss), [Base](https://github.com/liferay/clay/blob/master/packages/clay-css/src/scss/variables/_globals.scss), [Atlas](https://github.com/liferay/clay/blob/master/packages/clay-css/src/scss/atlas/variables/_globals.scss)
+
+There are also useful mixins and functions:
+
+- [Breakpoints](https://github.com/liferay/clay/blob/master/packages/clay-css/src/scss/bootstrap/mixins/_breakpoints.scss)
+- [Text-Truncate](https://github.com/liferay/clay/blob/master/packages/clay-css/src/scss/bootstrap/mixins/_text-truncate.scss)
+- [Deprecate](https://github.com/liferay/clay/blob/master/packages/clay-css/src/scss/bootstrap/mixins/_deprecate.scss)
+- [Clay Buttons](https://github.com/liferay/clay/blob/master/packages/clay-css/src/scss/mixins/_buttons.scss)
+- [Clay Grids](https://github.com/liferay/clay/blob/master/packages/clay-css/src/scss/mixins/_grids.scss)
+- [overflow-wrap, user-select, hyphens, placeholder](https://github.com/liferay/clay/blob/master/packages/clay-css/src/scss/mixins/_vendor-prefixes.scss)
+
+The second one is "overwrite mode" and it allow us to customize the general aspects of the framework and its components.
+
+In order to overwrite a variable of Clay, you just need to define its value before the import line
+
+---
+
+Let's say we need to change the `primary color` of our `custom-theme`.
+
+We are currently importing `atlas-variables` in our `main.scss`
+
+```scss
+//custom-theme-folder/main.scss
```

Comment:
We should mention including the variable here won't change the `$primary` color across all components that use `$primary`. This will only change the `$primary` color for the CSS inside `main.scss` which may or may not be what you want.

Reply:
allow -> allows